radar/web/src/map-info/index.ts

```diff
@@ -17,10 +17,11 @@ export const kRegisteredMaps: Record<string, () => Promise<LoadedMap>> = {
     cs_office: () => import("./cs_office").then((value) => value.default),
     de_ancient: () => import("./de_ancient").then((value) => value.default),
     de_anubis: () => import("./de_anubis").then((value) => value.default),
+    de_basalt: () => import("./de_basalt").then((value) => value.default),
     de_cache: () => import("./de_cache").then((value) => value.default),
     de_dust2: () => import("./de_dust2").then((value) => value.default),
+    de_edin: () => import("./de_edin").then((value) => value.default),
     de_inferno: () => import("./de_inferno").then((value) => value.default),
-    de_mills: () => import("./de_mills").then((value) => value.default),
```

Comment:
What's wrong with `de_mills`?

Reply:
De_mills is removed and replaced by basalt and thera